samples/MvcAngular/AngularClient/app.js

```diff
@@ -0,0 +1,5 @@
+var express = require("express");
```

Comment:
In the marvelous ASP.NET 5 world, are node.js apps common?
I wonder if we shouldn't use an ASP.NET 5 app here, with a basic `app.UseStaticFiles()`.